runtime_builders/publish_builders.py

```diff
@@ -55,18 +55,25 @@ def main():
 
 def _publish_latest(builder_dir):
     for f in glob.glob(os.path.join(builder_dir, '*.yaml')):
-        with open(f, 'r') as f:
-            config = yaml.safe_load(f)
-
-        latest = config['latest']
-        project_name = config['project']
-
-        parts = os.path.splitext(latest)
-        prefix = builder_util.RUNTIME_BUCKET_PREFIX + project_name + '-'
-        latest_file = parts[0].replace(prefix, '')
-        file_name = '{0}.version'.format(project_name)
-        full_path = builder_util.RUNTIME_BUCKET_PREFIX + file_name
-        builder_util.write_to_gcs(full_path, latest_file)
+        try:
+            with open(f, 'r') as f:
+                config = yaml.safe_load(f)
+
+            latest = config['latest']
+            project_name = config['project']
+
+            parts = os.path.splitext(latest)
+            prefix = builder_util.RUNTIME_BUCKET_PREFIX + project_name + '-'
+            latest_file = parts[0].replace(prefix, '')
+            file_name = '{0}.version'.format(project_name)
+            full_path = builder_util.RUNTIME_BUCKET_PREFIX + file_name
+            builder_util.write_to_gcs(full_path, latest_file)
+        except ValueError as ve:
```

Comment:
What would throw the value error and the keyerror? Could we narrow the try/catch?

Reply:
I'm not sure about the ValueError, I'll like take that out. The KeyError is for 

```python
latest = config['latest']
project_name = config['project']
```

Reply:
I'd replace the KeyError with an 'if 'latest' not in config or 'project' not in config'

Reply:
PTAL